core/version/gear.ex

```diff
@@ -26,6 +26,15 @@ defmodule AntikytheraCore.Version.Gear do
         {:error, {:already_loaded, ^gear_name}} -> :ok
       end
       L.info("successfully loaded '#{gear_name}' (#{latest_version})")
+      if :code.get_mode() != :interactive do
+        # Load module manually
+        Enum.each(Application.spec(gear_name, :modules), fn(mod) ->
+          case :code.load_file(mod) do
+            {:module, _}      -> :ok
+            {:error , reason} -> raise "Failed to load '#{mod}': #{inspect(reason)}"
```

Comment:
According to Erlang’s documentation, `reason` will be one of five atoms described [here](http://erlang.org/doc/man/code.html#data-types).
So, I think `inspect/1` is unnecessary.

Reply:
Fixed 74fffe3a984aa53c3a07def6a3cbebd67794932b